Paco-Server/src/com/google/sampling/experiential/server/reports/CloudSQLReportsDaoImpl.java

```diff
@@ -20,22 +20,28 @@
 import com.google.appengine.tools.cloudstorage.GcsOutputChannel;
 import com.google.common.collect.Lists;
 import com.google.sampling.experiential.datastore.EventServerColumns;
+import com.google.sampling.experiential.server.CloudSQLConnectionManager;
+import com.google.sampling.experiential.server.ExperimentAccessManager;
+import com.google.sampling.experiential.server.QueryConstants;
+import com.google.sampling.experiential.server.reports.CloudSQLReportsDao;
 import com.pacoapp.paco.shared.util.ErrorMessages;
 
 public class CloudSQLReportsDaoImpl implements CloudSQLReportsDao {
   public static final Logger log = Logger.getLogger(CloudSQLReportsDaoImpl.class.getName());
-  private static final String GET_PARTICIPANTS_QUERY = "select who from expwho where " + EventServerColumns.EXPERIMENT_ID+ " =?";
   
   @Override
-  public List<String> getParticipants(Long expId, String who) throws SQLException {
+  public List<String> getACLedParticipants(Long expId, String who) throws SQLException {
     List<String> userLst = null;
     Connection conn = null;
     PreparedStatement statementSelectParticipants = null;
     try {
-      if (who == null) {
+      List<Long> adminExperimentsinDB = ExperimentAccessManager.getExistingExperimentIdsForAdmin(who, 0,
```

Comment:
It looks like the method you want here is ExperimentAccessManager's 
    public static boolean isUserAdmin(Long experimentId, String loggedInUserEmail)